packages/frontend/src/components/Header.vue

```diff
@@ -42,4 +48,8 @@ header.v-app-bar.v-toolbar {
   box-shadow: none;
   background-color: var(--vscode-editor-background, #1e1e1e) !important;
 }
+header.v-app-bar.v-toolbar .v-icon {
+  padding: 9px 5px 6px;
+  opacity: 0.55;
+}
```

Comment:
consider to get approved from UX team (Cohen-Zur, Nirit)
consider to use a 'rem' units instead of 'px' ones

Reply:
OK but existing code is using px.